src/__tests__/index.test.ts

```diff
@@ -114,6 +114,13 @@ const vectors: Array<TestVector> = [
       { text: 'cxa4524257b3511fb9574009785c1f1e73cf4097e7', hex: '01a4524257b3511fb9574009785c1f1e73cf4097e7' },
     ],
   },
+  {
+    name: 'XVG',
+    coinType: 77,
+    passingVectors: [
+      { text: 'D7MKQnLxXEqn84PN42jWAVhvrXEuULLV9r', hex: '76a914183ffcc41f3095bea7ff324e52a65b46c74126e188ac' },
```

Comment:
Where did you find this test data? Please specify source (eg: test code, block explorer).

Reply:
in JSON section 
https://vergeexplorer.com/tx/881facd64fc4a2d56b3b76598d45ff51abcc29028220281fafa9247a46a7ecb0
![verge](https://user-images.githubusercontent.com/58694244/100514999-c1fc1f00-3181-11eb-8ff2-3801f9afcf4b.png)